### PR TITLE
Added more to the readme and tweaked the ps script for a less painful setup in DK Signal

### PR DESCRIPTION
**Pull request by carrollh**

### CODE REVIEW

**Review by srazin — APPROVED** [review]
looks good please go ahead and merge whenever ready.

File: README.md [hunk 1]
```diff
@@ -23,10 +23,38 @@ The arguments field on the New Action panel should be given the following:
...
 [Signal iQ Configuration](../master/Signal_iQ.webm)
 
 Click the link (and then 'View Raw')for a video configuration walkthrough of Windows Signal:
 
 [Windows Signal Configuration](../master/Windows_Signal.webm)
+
+The following PowerShell code will configure a Task after prompting for all the required inputs:
+
+```
+# prompt user for important paths
```
Comment (srazin) [inline README.md:38]: Question. Why not create a file that captures that? This way it can be versioned, etc. as the rest of the code.

Comment (carrollh) [reply README.md:38]: We can do that if you'd rather.

**Comment by srazin** [comment]
I think that would not be a bad idea. Let's do that and point in README to
state that use that thing.

-----
Regards,
Sergey A. Razin, Ph.D. |  Chief Technology Officer
SIOS Technology Corp.
P: 650.669.8914
E: sergey.razin@us.sios.com
LinkedIn: http://www.linkedin.com/in/sergeyrazin
Twitter @ TechDozor <https://twitter.com/techdozor>
www.us.sios.com

**Pull request merged**